templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -462,13 +462,24 @@
               {
                 "Variable": "$.wasReconciled",
                 "BooleanEquals": true,
-                "Next": "Do nothing"
+                "Next": "Publish reconciliation update to dr2-notifications"
               }
             ],
             "Default": "Throw Reconciler job error"
           },
-          "Do nothing": {
-            "Type": "Pass",
+          "Publish reconciliation update to dr2-notifications": {
+            "Type": "Task",
+            "Resource": "arn:aws:states:::sns:publish",
+            "Parameters": {
+              "Message": {
+                "reconciliationUpdate": "Asset was reconciled",
+                "assetId.$": "$.AssetInfo.assetId",
+                "messageId": "$.messageId",
```

Comment:
Where does this UUID come from? Is there any benefit to us using `"field.$": "States.UUID()"` instead ([link](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-intrinsic-functions.html#asl-intrsc-func-uuid-generate))?

Reply:
Because you wrote in the "DA-ADR-13 Notification Message Tracing" doc that "All new notification message schemas should include `messageId`, `parentMessageId`, and `executionId`...". I thought these 3 would be in the `message` and have to be extracted

Reply:
`messageId` in the message we receive would become `parentMessageId` in the message we send. `executionId` in our case will be the Step Function execution name.